src/index.js

```diff
@@ -17,9 +17,6 @@ class RecPlayer extends Component {
     };
   }
   componentWillReceiveProps(nextProps) {
-    this.setState({
```

Comment:
I don't understand why this was removed

Reply:
It was removed because it didn't work and introduced a bug and I'm not sure why it is there in the first place. Setting the state like that does not actually stop or start the playback, instead you should call the playPause-function. That state property is only used to determine if the controls on the ui should show play or pause icon, so this resulted in the player showing the wrong icon in the controls. Tell me if this somehow broke some functionality on elma.online that I'm not aware of. Anyway I think this recplayer component code is not that easy to understand and should be rewritten in modern react.

Reply:
autoplay on elmaonline seems unaffected

Reply:
Cool, makes sense